src/legacy/ui/public/chrome/directives/_kbn_chrome.scss

```diff
@@ -5,7 +5,6 @@
  */
 // SASSTODO: Naming here is too embedded and high up that changing them could cause major breaks
 #kibana-body {
-  overflow-x: hidden;
```

Comment:
Unfortunately, we can't make major changes like this to the content wrapper of Kibana plugins. This wrapper affects all plugins and could cause major layout breaks.

I have looked into other Kibana plugins that use the EuiDragDrop component, like Maps, and they don't seem to have the same problem or they've been able to overcome it in a different way:

![Screen Recording 2019-11-21 at 11 02 AM](https://user-images.githubusercontent.com/549577/69354847-c6c4f380-0c4e-11ea-9dc8-4c196c8c148a.gif)

I'd encourage you to find a solution that wouldn't affect every plugin within Kibana.

Reply:
Then maybe Elastic may want to add the solution to the Eui docs to help all plugin developers https://elastic.github.io/eui/#/display/drag-and-drop

Reply:
Because the current examples don't work in custom plugins